protocol.html

```diff
@@ -509,59 +509,62 @@ <h3 property="schema:name"><span>Issue</span>: Solid Notification Vocabulary and
-                    <dt about="#notify-type" property="skos:prefLabel" typeof="skos:Concept"><dfn id="notify-type">type</dfn></dt>
-                    <dd about="#notify-type" property="skos:definition"><code>type</code> predicate (<code>rdf:type</code>) denotes the subscription type (<a href="#subscription" rel="rdfs:seeAlso">Subscription</a>).</dd>
-                  </dl>
+                      <p about="" id="server-link-storage-notification-channel" rel="spec:requirement" resource="#server-link-storage-notification-channel"><span property="spec:statement">When a server wants to enable applications to discover <a href="#notification-channel">Notification Channels</a> available to a <a href="https://solidproject.org/TR/protocol#storage">storage</a> in which a given <a href="https://solidproject.org/TR/protocol#resource">resource</a> is in, the <span rel="spec:requirementSubject" resource="spec:Server">server</span> <span rel="spec:requirementLevel" resource="spec:MUST">MUST</span> advertise the associated resources describing the channel information by responding to an HTTP request including a <code>Link</code> header with the <code>rel</code> value of <code>http://www.w3.org/ns/solid#storageDescription</code> [<cite><a class="bibref" href="#bib-solid-protocol">SOLID-PROTOCOL</a></cite>] and the Description Resource as link target [<cite><a class="bibref" href="#bib-rfc8288">RFC8288</a></cite>].</span></p>
 
-                  <div class="issue" id="discovery-subscription-metadata" rel="schema:hasPart" resource="#discovery-subscription-metadata">
-                    <h4 property="schema:name"><span>Issue</span>: Notification Subscription Metadata Discovery</h4>
-                    <div datatype="rdf:HTML" property="schema:description">
-                      <p>The specific mechanism defining the discovery of the <a href="#notification-subscription-metadata-resource">Notification Subscription Metadata Resource</a> is under consideration: <a href="https://github.com/solid/notifications/issues/58" rel="cito:citesAsRelated">issues/58</a>.</p>
+                      <p>
+                        <span about="" id="server-notification-channel-resource-accept" rel="spec:requirement" resource="#server-notification-channel-resource-accept">
+                          <span property="spec:statement">
+                            <span rel="spec:requirementSubject" resource="spec:Server">Servers</span>
+                            <span rel="spec:requirementLevel" resource="spec:MUST">MUST</span> accept requests on the Notification Channel Resource when the value of the <code>Accept</code> header indicates a preferred representation in <code>application/ld+json</code> [<cite><a class="bibref" href="#bib-json-ld11">JSON-LD11</a></cite>].
```

Comment:
Is it necessary to require JSON-LD 1.1, or can we just say JSON-LD?

Reply:
I think JSON-LD will suffice and generally entails the latest published version.

But this reminds me to mention something related about the JSON-LD form. Should mention requiring https://www.w3.org/TR/json-ld11/#compacted-document-form in favour of the details on the context value being array currently.

Reply:
We know the spec works with JSON-LD11 so we should require and reference that specifically.

I will update the references to JSON-LD11 throughout the document separately - not part of this PR.

I can also follow-up on any changes to context and document-form in a separate PR. Discussion here or elsewhere can continue.

Reply:
> Should we require https://www.w3.org/TR/json-ld11/#compacted-document-form instead of @context with an array?

A document with `"@context": [ ... ]` _is_ compacted document form